State.cpp

```diff
@@ -17,6 +17,14 @@ std::string State::getName()
 StateType State::getType()
 {
     return this->type;
+}   
+
+std::string State::getStringType() 
+{  
+    if (this->type == 0) {return "normal"}   
+    else if (this->type == 1) {return "acccept"}  
+    else if (this->type == 2) {return "reject"}  
+    else if (this->type == 3) {return "start"}  
```

Comment:
forgot `;` at the end of the return statements.
Also, line 25 reads `acccept` with 3 c's